docs/reference/configuration-reference/share-settings.md

```diff
@@ -0,0 +1,34 @@
+---
+navigation_title: "Share settings"
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/share-settings-kb.html
+applies_to:
+  deployment:
+    ess: ga 9.1, ga 8.19
+    self: ga 9.1, ga 8.19
+---
+
+# Share settings in {{kib}} [share-settings-kb]
+
+Configure share settings in your `kibana.yml` configuration file.
+Share settings allow you to customize behavior related to URL sharing in Kibana.
+
+## URL Expiration settings [url-expiration-settings]
+
+URL expiration settings control the behavior of unused URLs cleanup background task, which runs using Task Manager plugin. This task allows you to periodically cleanup saved objects of type `url` that have not been accessed in the specified period of time, specified by `share.url_expiration.duration` config option. Each saved object is a representation of a URL generated through the share functionality. Those settings are disabled by default. You need to manually configure them in order to use this feature.
+
+::::{warning}
+We do not recommend enabling this on lower tier Elasticsearch instances as it can lead to memory spikes and instability. The minimum recommend Elasticsearch instance to use the default setting should have 2 GB RAM. Lower tier instances shouldn’t set `share.url_expiration.url_limit` to more than `1000`.
```

Comment:
```suggestion
We do not recommend enabling this on lower tier Elasticsearch instances as it can lead to memory spikes and instability. The Elasticsearch instance should have at least 2 GB RAM to use the default setting. Lower tier instances shouldn’t set `share.url_expiration.url_limit` to more than `1000`.
```